content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -0,0 +1,166 @@
+
+.. _belgium/introduction:
+
+Fiscal certification: POS restaurant
+====================================
+
+In Belgium, the owner of a cooking business such as a restaurant or food truck is required by law to
+use a Government-certified **Cash Register System** for their receipts. This applies if their yearly
+earnings (excluding VAT, drinks, and take-away food) exceed 25,000 euros.
+
+This Government-certified system entails the use of a **certified POS system**, along with a device
+called a **Fiscal Data Module** (or **black box**) and a **VAT Signing Card** (delivered after
+registration by the Service Public Federal Finances).
+
+A certified POS system
+----------------------
+
+The Odoo POS system is certified for Odoo 9 (**Odoo Online**) and Odoo 13 to Odoo 15 (**Odoo
+Online and odoo.sh**) to ensure full compliance with the law.
+
+A certified POS system must adhere to rigorous Government regulations, which means it operates
+differently from a non-certified POS.
+
+- You cannot:
+
+  - Make a refund.
+  - Set up and use the **global discounts** feature (the `pos_discount` module is blacklisted and
+    cannot be activated).
+  - Set up and use the **loyalty programs** feature (the `pos_loyalty` module is blacklisted and
+    cannot be activated).
+  - Reprint receipts (the `pos_reprint` module is blacklisted and cannot be activated).
+  - Modify prices in order lines.
+  - Modify or delete order lines in POS orders.
+  - Sell products without a valid VAT number.
+  - Use a POS that is not connected to an IoT box.
+
+- The :doc:`cash rounding <../../sales/point_of_sale/shop/cash_rounding>` feature must be
+  activated and set to a :guilabel:`Rounding Precision` of `0,05` and a :guilabel:`Rounding Method`
+  set as :guilabel:`Half-Up`.
+- Taxes must be set as included in the price. :menuselection:`Point of Sale --> Configuration -->
+  Settings --> Taxes section --> External link --> Advanced Options` and enable :guilabel:`Included
+  in Price`.
+- At the start of a POS session, users must click :guilabel:`Work in` to clock in. Doing so allows
+  the registration of POS orders. If users are not clocked in, they cannot make POS orders.
+  Likewise, they must click :guilabel:`Work Out` to clock out at the end of the session.
+
+.. warning::
+   - Odoo :ref:`On-premise <belgium/on-premise>` is *not* certified.
+   - If you configure a POS to work with a black box, you cannot use it again without the black box.
+
+The Fiscal Data Module
+----------------------
+
+The :abbr:`FDM (Fiscal Data Module)`, or `black box
+<https://www.systemedecaisseenregistreuse.be/systemes-certifies>`_, is a Government-certified device
+that works together with the Point of Sale application and saves your POS orders information.
+Concretely, a **Hash** (:dfn:`unique code`) is generated for each POS order and added to its
+receipt. This allows the Government to verify that all revenue is declared.
+
+.. note::
+   - We recommend the use of the Cleancash SC-B.
```

Comment:
We don't recommand cleancash, it's not working. 
We only support the blackbox of boitenoire.